src/audio/pipeline.c

```diff
@@ -258,6 +257,8 @@ int pipeline_free(struct pipeline *p)
 static int pipeline_comp_params(struct comp_dev *current, void *data, int dir)
 {
 	struct pipeline_data *ppl_data = data;
+	int dl = current->pipeline->ipc_pipe.deadline;
+	int dl_m = current->pipeline->ipc_pipe.deadline - 1;
```

Comment:
care to expand the variable names into something more meaningful.

Reply:
removed the variables and add comment